tests/integration/test_orders_cli.py

```diff
@@ -649,6 +666,33 @@ def test_cli_orders_request_tools(invoke, geom_geojson):
             "product_bundle": "analytic",
         }],
         "tools":
-        tools_json
+        tools_json,
+        "metadata":
+        stac_json
+    }
+    assert order_request == json.loads(result.output)
+
+
+@respx.mock
+def test_cli_orders_request_stac(invoke, stac_json):
+
+    result = invoke([
+        'request',
+        '--name=test',
+        '--id=4500474_2133707_2021-05-20_2419',
+        '--bundle=analytic',
+        '--item-type=PSOrthoTile'
+    ])
+
+    order_request = {
+        "name":
+        "test",
+        "products": [{
+            "item_ids": ["4500474_2133707_2021-05-20_2419"],
+            "item_type": "PSOrthoTile",
+            "product_bundle": "analytic",
+        }],
+        "metadata":
+        stac_json
```

Comment:
I'm assuming the API treats `"metadata": {}` the same as omitting the `metadata` member altogether - but don't know this for sure.

Reply:
In these tests we're only testing for when `--no-stac` is set to false, i.e., we are requesting STAC metadata. I'm not entirely sure what would happen if we requested `"metadata": {}`. I suppose this should be something we test for in `tests.integration.test_order_request.test_build_request()`. Nice catch.

Reply:
Indeed, by adding a `--no-stac` flag, the output from the Orders API does not have `"metadata": {}`. See: https://github.com/planetlabs/planet-client-python/pull/676/commits/84d2e5423afa563005678e770d9f5aba2a7d2adb